src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -306,22 +306,23 @@ public void registerUserForCourse(User user, Course course) {
      *
      * @param exerciseIds the ids to get the active students for
      * @param periodIndex the deviation from the current time
+     * @param length the length of the chart which we want to fill. This can either be 4 for the course overview or 8 for the courde detail view
```

Comment:
I think the "8" should be "16"
(this 8 instead of 16 is repeated in multiple other comments)
(Most likely you noticed during development that 8 are not enough and wanted to increase to 16 but forgot to change the comments ? 😅)

Reply:
That's true, forgot to adapt the comments. Done in 9472a29